app/src/App.tsx

```diff
@@ -62,6 +60,14 @@ const App: React.FC = (props: any) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [ethereum])
 
+  useEffect(() => {
+    window.onerror = function(errorMsg) {
+      if (errorMsg) {
+        windowObj.location.reload(true)
+      }
+    }
+  }, [network, windowObj.location])
+
```

Comment:
also I don't think this is good practice to catch error like this and reloading page....